proto/cosmos/feegrant/v1beta1/query.proto

```diff
@@ -21,6 +21,11 @@ service Query {
   rpc Allowances(QueryAllowancesRequest) returns (QueryAllowancesResponse) {
     option (google.api.http).get = "/cosmos/feegrant/v1beta1/allowances/{grantee}";
   }
+
+  // IssuedAllowances returns all the grants given by an address
+  rpc IssuedAllowances(QueryIssuedAllowancesRequest) returns (QueryIssuedAllowancesResponse) {
+    option (google.api.http).get = "/cosmos/feegrant/v1beta1/issued/{granter}";
```

Comment:
I'm not sure I like the `IssuedAllowances` name. It's sounds confusing with `Allowances`.

Maybe `AllowancesByGranter` to be more explicit

Reply:
Would we want to change `Allowances` to `AllowancesByGrantee` or leave it as is?

Reply:
That would be proto-breaking, so no, though it would be the ideal choice.

I still prefer `Allowances+AllowancesByGranter` than `Allowances+IssuedAllowances`

Reply:
Ok I will make the change